flake.nix

```diff
@@ -35,6 +35,7 @@
       supportedSystems = [
         "x86_64-linux"
         "x86_64-darwin"
+        "aarch64-darwin"
       ];
```

Comment:
Is it possible to also add aarch64-linux to CI?

Reply:
Thanks for the feedback! Based on your suggestion, I'm adding that in a separate PR #1612